common/tools/dev-tool/package.json

```diff
@@ -22,7 +22,7 @@
     "prebuild": "npm run clean",
     "unit-test": "mocha --require ts-node/register test/**/*.spec.ts",
     "build:samples": "echo Skipped.",
-    "docs": "typedoc --excludePrivate --excludeNotExported --mode file --out ./dist/docs ./src"
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --mode file --out ./dist/docs ./src"
```

Comment:
Can you provide an example of the difference this brings in?
(If possible, screenshots may help.)

Reply:
@HarshaNalluru Does https://github.com/Azure/azure-sdk-for-js/issues/12948#issuecomment-748383084 help?

Reply:
Hmm, thanks, I can imagine.